bin/inbox-api.py

```diff
@@ -45,19 +45,14 @@
     default=True,
     help="Also start the syncback service",
 )
-@click.option(
-    "--enable-tracer/--disable-tracer",
-    default=True,
-    help="Disables the stuck greenlet tracer",
-)
```

Comment:
Tracer is gevent specific and this code is not shared with anything else as APIs already run in threads in production, so I just removed this option.